meta/runtime.yml

```diff
@@ -1,79 +1,2 @@
 ---
-requires_ansible: ">=2.13"
```

Comment:
We should not remove this , instead we should tombstone it.. 
Please see the details [here ](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#tombstoning-a-module-or-plugin-in-a-collection)

Reply:
Done